spec/lib/form1010_ezr/service_spec.rb

```diff
@@ -65,6 +50,29 @@ def submit_form(form)
           )
         end
       end
+
+      context 'when the form includes a Mexican province' do
+        let(:form) { get_fixture('form1010_ezr/valid_form_with_mexican_province') }
+
+        it "overrides the original province 'state' with the correct province initial and renders a " \
+           'successful response', run_at: 'Mon, 23 Oct 2023 23:42:13 GMT' do
+          VCR.use_cassette(
+            'form1010_ezr/authorized_submit_with_mexican_province',
+            match_requests_on: [:method]
```

Comment:
why is `match_requests_on` set to method? all that tests is that a POST request was made

Reply:
Fixed!